FreeRTOS/iot-reference-arm-corstone3xx
# Build and debug FRI using VS Code

Description
-----------
Added VS Code tasks and launch configuration for building and debugging FRI
Restructured docs folder for better user experience
Add short options to build and run scripts

Test Steps
-----------
Tested the changes in the CI, no regression observed.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
